assets/js/app.js

```diff
@@ -0,0 +1,9 @@
+$('.start').html(AppTemplates.start(heroes));
+
+$('.start').click(function() {
```

Comment:
You will want to listen for a `.start-pic` to be clicked within the `.game` div: Look at the optional `selector` that `jQuery.on` can take: http://api.jquery.com/on/